articles/why-nodejs-good-for-online-store/index.md

```diff
@@ -0,0 +1,38 @@
+
+**Non-Blocking Code Execution –** Node.js platform is built to handle asynchronous I/O and manage the process without any delay, which means it does away with the possibility of any delay in execution. It is not the case compared to synchronous code execution where execution is sequential, meaning one thread cannot execute unless the previous one is finished. It would bring delay in overall execution and hence more response time. In an e-commerce store, each customer request should be handled simultaneously instead of sequentially as many clients are accessing the store simultaneously. Node.js handles this correctly since it allows simultaneous processing.
+
+### Platforms that use Node.js to build an E-Commerce Store
+Here are the few ready solutions that create E-commerce app:
+
+**Prime Fusion** – This is based on a JavaScript framework that includes MongoDB, Express, Angular.js, Node.js, GraphQL, webpack React, Gulp, Babel Mocha. The databases and frameworks mentioned above provide the developers with the required tools to build an online store. Prime Fusion does not require special knowledge about building a store since it is backed with the framework.
+
+**Reaction Commerce** – This platform is built on the Meteor framework, a fast, flexible, and scalable solution that works well with npm, Docker, and React. The platform codebase is modular, meaning that it can be customized to meet the developer's needs. The platform provides real-time analytics to assist the users in making better business decisions. 
+
+**Trader io** – This is also built on the MEAN stack. It offers flexible and scalable API and has basic functionalities e-commerce store needs. The developers had previously created several e-commerce applications, and they took the existing code and made it open source.
+
+### What to consider on whether to use Node.js
+A few factors can be considered on whether to use Node.js as a preferred development solution. Some of them include:
+
+**Choosing the right solution** - They are ready-made solutions that use Node.js to choose from, as mentioned above. It depends on the user's needs, the number of features the system should have, and its complexity. Also, one can build the app from scratch depending on the unique feature the app is intended to have that ready-made solutions are missing.
+
+**Technology Coherence** - Node.js only serves as a backend for an e-commerce store; other technologies can be considered when building other application parts to avoid any problems that may appear when developing the app. It is better to use a ready package offered by e-commerce platforms, as earlier discussed, to ensure that all the frameworks and solutions remain consistent.
```

Comment:
In this sentence `It is better to use a ready package offered by e-commerce platforms ...`  replace `ready` with `existing`